### PR TITLE
chore: polish kb52-62 files

### PR DESCRIPTION
**Pull request by Meng-Heng**

### CODE REVIEW

File: knowledge-base/kb0052.md [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
+- Close Keyman.
+- Go to Windows Control Panel.
+- Select 'Add/Remove Programs' or 'Programs & Features'.
+- Find Tavultesoft Keyman Desktop in the list.
+- Select **Change**.
+- Click **Next** and select **Change** again.
+- Expand the Addins item.
+- Click **Text Services Framework**.
+- Select 'Entire feature will be unavailable'.
+- Click **Next** then **Change** to complete the process.
```
Comment (Meng-Heng) [inline knowledge-base/kb0052.md:17]:
```suggestion
1. Close Keyman.
2. Go to Windows Control Panel.
3. Select 'Add/Remove Programs' or 'Programs & Features'.
4. Find Tavultesoft Keyman Desktop in the list.
5. Select **Change**.
6. Click **Next** and select **Change** again.
7. Expand the Addins item.
8. Click **Text Services Framework**.
9. Select 'Entire feature will be unavailable'.
10. Click **Next** then **Change** to complete the process.

```

File: knowledge-base/kb0053.md [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
+Follow the steps below to unbundle a keyboard layout which has come bundled to an older version of Keyman. This prevents having to install the older version of Keyman in order to gain access to the keyboard layout.
+
+### Unbundling a keyboard layout from a Keyman 5 or Keyman 6 bundled installer-
+
+- Start the bundled installer.
+- From the initial setup screen, click **About** and leave the installer open on this screen.
+- Open the Windows Start menu, click Run, type **%temp%**, and press enter.
+- In the Explorer window which opens, look for a folder with a name that begins with the letters **kmt** and ends with a few random letters and numbers. Open that folder.
+- In that folder you will find one or more **.kmp** files. Copy the **.kmp** file(s) to another folder on your computer.
+- Return to the Keyman 6 bundled installer and exit it.
```
Comment (Meng-Heng) [inline knowledge-base/kb0053.md:15]:
```suggestion
1. Start the bundled installer.
2. From the initial setup screen, click **About** and leave the installer open on this screen.
3. Open the Windows Start menu, click Run, type **%temp%**, and press enter.
4. In the Explorer window which opens, look for a folder with a name that begins with the letters **kmt** and ends with a few random letters and numbers. Open that folder.
5. In that folder you will find one or more **.kmp** files. Copy the **.kmp** file(s) to another folder on your computer.
6. Return to the Keyman 6 bundled installer and exit it.  

```

File: knowledge-base/kb0054.md [hunk 3]
```diff
@@ -1,18 +1,15 @@
...
- * Keyman Desktop Professional 7.0
+# HOWTO: Deselect the Text Services Framework Addin when using non-Unicode keyboard layouts
+
+### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant
+
+You may need to deselect the Text Services Framework Addin when working with legacy, non-Unicode, keyboard layouts on newer versions of Keyman Desktop and Microsoft Windows. If you rely heavily on such keyboards, an alternative solution is to uninstall the Keyman TSF Addin entirely (see [KMKB0052](/kb/?id=52)).
+
+To deselect the Text Services Framework Addin for a given keyboard layout:
+- In Keyman, select your desired Keyman keyboard layout.
+- From the Windows language bar, switch your keyboard input from **Tavultesoft Keyman Desktop TSF Addin** to the Windows keyboard input option.
```
Comment (Meng-Heng) [inline knowledge-base/kb0054.md:9]:
```suggestion
1. In Keyman, select your desired Keyman keyboard layout.
2. From the Windows language bar, switch your keyboard input from **Tavultesoft Keyman Desktop TSF Addin** to the Windows keyboard input option.

```

File: knowledge-base/kb0057.md [hunk 4]
```diff
@@ -1,30 +1,28 @@
...
+### Solution 2
+ Disable '/' shortcuts -- In the Microsoft Office Excel menu, disable the use of '/' as a shortcut key.
+In Excel 2007: 
+ 
+  - Open the Office menu button in Excel
+  - Click **Excel Options**
+  - Select **Advanced**
+  - Scroll down to **Lotus Compatibility**
+  - Delete '/' in the text box beside **Microsoft Office Excel <ins>m</ins>enu key:**
+  - Click **OK**
```
Comment (Meng-Heng) [inline knowledge-base/kb0057.md:20]:
```suggestion
  1. Open the Office menu button in Excel
  2. Click **Excel Options**
  3. Select **Advanced**
  4. Scroll down to **Lotus Compatibility**
  5. Delete '/' in the text box beside **Microsoft Office Excel <ins>m</ins>enu key:**
  6. Click **OK**

```

File: knowledge-base/kb0059.md [hunk 5]
```diff
@@ -1,48 +1,32 @@
...
+In these instructions, we are using the SIL IPA Unicode keyboard and linking it to the Icelandic language.  Substitute these names accordingly when you are configuring your own keyboards.
+
+- (Desktop Pro only) Remove the Keyman Desktop Text Services Framework Add-in.  Toolbox does not work with the Text Services Framework correctly.  To remove the Add-in:
+   - Start **Control Panel**, **Add/Remove Programs** (this is now called **Programs and Features** in Vista)
+   - Find **Tavultesoft Keyman Desktop** and click **Change**.
+   - Click **Next**, then **Change** in the install wizard that appears.
+   - Expand the **Addins** node and change the setting for **Text Services Framework Addin** to **Uninstall** or **Not Installed**
+   - Click **Next** to apply the changes
+   - Note: until you restart Windows, the Addin may still be visible in some aspects 
+       of the Windows user interface.
```
Comment (Meng-Heng) [inline knowledge-base/kb0059.md:16]:
```suggestion
   i. Start **Control Panel**, **Add/Remove Programs** (this is now called **Programs and Features** in Vista)
   ii. Find **Tavultesoft Keyman Desktop** and click **Change**.
   iii. Click **Next**, then **Change** in the install wizard that appears.
   iv. Expand the **Addins** node and change the setting for **Text Services Framework Addin** to **Uninstall** or **Not Installed**
   v. Click **Next** to apply the changes
   vi. Note: until you restart Windows, the Addin may still be visible in some aspects of the Windows user interface.

```

File: knowledge-base/kb0059.md [hunk 6]
```diff
@@ -1,48 +1,32 @@
...
+
+- Add the SIL IPA Unicode Keyman keyboard as a System Shadow Keyboard (the SIL IPA Unicode Keyman keyboard must be installed for all users - if it isn't you must uninstall the SIL IPA Unicode Keyman keyboard and reinstall it for all users - this is a Windows requirement in order for the SIL IPA Unicode Keyman keyboard to be visible in the list of Windows keyboards):
+    - Right-click on the **Language Bar** and select **Settings**.
+    - Click Add, select **Icelandic**, expand the **Keyboards** node, and select **Show More**.
+    - Select **SIL IPA Unicode** from the expanded list of keyboards and click **OK**.
+    - Click **OK** again to save changes.
```
Comment (Meng-Heng) [inline knowledge-base/kb0059.md:22]:
```suggestion
    i. Right-click on the **Language Bar** and select **Settings**.
    ii. Click Add, select **Icelandic**, expand the **Keyboards** node, and select **Show More**.
    iii. Select **SIL IPA Unicode** from the expanded list of keyboards and click **OK**.
    iv. Click **OK** again to save changes.  

```

File: knowledge-base/kb0059.md [hunk 7]
```diff
@@ -1,48 +1,32 @@
...
+    - Right-click on the **Language Bar** and select **Settings**.
+    - Click Add, select **Icelandic**, expand the **Keyboards** node, and select **Show More**.
+    - Select **SIL IPA Unicode** from the expanded list of keyboards and click **OK**.
+    - Click **OK** again to save changes.  
+
+- In Toolbox, configure your language settings:
+   - Click **Project**, **Language Encodings**
+   - Select the language you wish to link, and click **Modify**.
+   - Click **Options**, and in the **Keyboard** combo box, select **SIL IPA Unicode**
+   - Click **OK**, **Close** to save changes.
```
Comment (Meng-Heng) [inline knowledge-base/kb0059.md:28]:
```suggestion
   i. Click **Project**, **Language Encodings**
   ii. Select the language you wish to link, and click **Modify**.
   iii. Click **Options**, and in the **Keyboard** combo box, select **SIL IPA Unicode**
   iv. Click **OK**, **Close** to save changes.

```

File: knowledge-base/kb0061.md [hunk 8]
```diff
@@ -1,64 +1,48 @@
...
+
+![Language Bar - Floating](assets/kb0061/languagebar_full.png)
+
+- Click on the Keyman button on the Language Bar, and select an alternative keyboard layout (in the picture, the alternative layout is US):
+
+![Language Bar - Floating - Menu](assets/kb0061/languagebar_full_menu.png)
+
+- When the Keyman Text Services Framework Addin is off, the Language Bar should look similar to the picture below:
+
+![Language Bar - Keyman TSF Addin Off](assets/kb0061/languagebar_full_off.png)
```
Comment (Meng-Heng) [inline knowledge-base/kb0061.md:20]:
```suggestion
1. Locate the Language Bar.  It is floating at the top of your screen.  See the picture below for an example of what the Language Bar looks like.

![Language Bar - Floating](assets/kb0061/languagebar_full.png)

2. Click on the Keyman button on the Language Bar, and select an alternative keyboard layout (in the picture, the alternative layout is US):

![Language Bar - Floating - Menu](assets/kb0061/languagebar_full_menu.png)

3. When the Keyman Text Services Framework Addin is off, the Language Bar should look similar to the picture below:

![Language Bar - Keyman TSF Addin Off](assets/kb0061/languagebar_full_off.png)

```

File: knowledge-base/kb0061.md [hunk 9]
```diff
@@ -1,64 +1,48 @@
...
+
+![Language Bar - On Taskbar](assets/kb0061/languagebar_min.png)
+
+- Click on the Keyman button on the Language Bar, and select an alternative keyboard layout (in the picture, the alternative layout is US):
+
+![Language Bar - On Taskbar - Menu](assets/kb0061/languagebar_min_menu.png)
+
+- When the Keyman Text Services Framework Addin is off, the Language Bar should look similar to the picture below:
+
+![Language Bar - Keyman TSF Addin Off](assets/kb0061/languagebar_min_off.png)
```
Comment (Meng-Heng) [inline knowledge-base/kb0061.md:34]:
```suggestion
1. Locate the Language Bar.  It is located on your Taskbar.  See the picture below for an example of what the Language Bar looks like.

![Language Bar - On Taskbar](assets/kb0061/languagebar_min.png)

2. Click on the Keyman button on the Language Bar, and select an alternative keyboard layout (in the picture, the alternative layout is US):

![Language Bar - On Taskbar - Menu](assets/kb0061/languagebar_min_menu.png)

3. When the Keyman Text Services Framework Addin is off, the Language Bar should look similar to the picture below:

![Language Bar - Keyman TSF Addin Off](assets/kb0061/languagebar_min_off.png)

```

File: knowledge-base/kb0062.md [hunk 10]
```diff
@@ -1,25 +1,19 @@
...
+# HOWTO: Find fonts that work with your Keyman Desktop keyboard
+
+### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant
+
+Keyman Desktop's Font Helper tool will help you find fonts that work with your language. Here's how:
+  - Open Keyman Desktop.
+  - Open the Keyman menu (from the '**K**' icon on the Windows taskbar, near the clock).
+  - Click **On Screen Keyboard**. This opens the On Screen Keyboard toolbar.
+  - Click the Font Helper icon on the toolbar (hover over the icons for their names).
+  - Select the keyboard you want to use, from the On Screen Keyboard toolbar or the Keyman menu.
```
Comment (Meng-Heng) [inline knowledge-base/kb0062.md:10]:
```suggestion
  1. Open Keyman Desktop.
  2. Open the Keyman menu (from the '**K**' icon on the Windows taskbar, near the clock).
  3. Click **On Screen Keyboard**. This opens the On Screen Keyboard toolbar.
  4. Click the Font Helper icon on the toolbar (hover over the icons for their names).
  5. Select the keyboard you want to use, from the On Screen Keyboard toolbar or the Keyman menu.

```

**Review by Meng-Heng — COMMENTED** [review]
I mixed up <ol> with <ul> and these are the changes I will apply.

File: knowledge-base/kb0059.md [hunk 11]
```diff
@@ -1,48 +1,34 @@
...
+
+In these instructions, we are using the SIL IPA Unicode keyboard and linking it to the Icelandic language.  Substitute these names accordingly when you are configuring your own keyboards.
+
+- (Desktop Pro only) Remove the Keyman Desktop Text Services Framework Add-in.  Toolbox does not work with the Text Services Framework correctly.  To remove the Add-in:
+   i. Start **Control Panel**, **Add/Remove Programs** (this is now called **Programs and Features** in Vista)
+   ii. Find **Tavultesoft Keyman Desktop** and click **Change**.
+   iii. Click **Next**, then **Change** in the install wizard that appears.
+   iv. Expand the **Addins** node and change the setting for **Text Services Framework Addin** to **Uninstall** or **Not Installed**
+   v. Click **Next** to apply the changes
+   vi. Note: until you restart Windows, the Addin may still be visible in some aspects of the Windows user interface.
```
Comment (darcywong00) [inline knowledge-base/kb0059.md:15]: Might need to use numbers to get the sublist to format

```suggestion
- (Desktop Pro only) Remove the Keyman Desktop Text Services Framework Add-in.  Toolbox does not work with the Text Services Framework correctly.  To remove the Add-in:
  1. Start **Control Panel**, **Add/Remove Programs** (this is now called **Programs and Features** in Vista)
  2. Find **Tavultesoft Keyman Desktop** and click **Change**.
  3. Click **Next**, then **Change** in the install wizard that appears.
  4. Expand the **Addins** node and change the setting for **Text Services Framework Addin** to **Uninstall** or **Not Installed**
  5. Click **Next** to apply the changes
  6. Note: until you restart Windows, the Addin may still be visible in some aspects of the Windows user interface.
```

File: knowledge-base/kb0059.md [hunk 12]
```diff
@@ -1,48 +1,34 @@
...
+    ii. Click Add, select **Icelandic**, expand the **Keyboards** node, and select **Show More**.
+    iii. Select **SIL IPA Unicode** from the expanded list of keyboards and click **OK**.
+    iv. Click **OK** again to save changes.  
+
+
+- In Toolbox, configure your language settings:
+   i. Click **Project**, **Language Encodings**
+   ii. Select the language you wish to link, and click **Modify**.
+   iii. Click **Options**, and in the **Keyboard** combo box, select **SIL IPA Unicode**
+   iv. Click **OK**, **Close** to save changes.
```
Comment (darcywong00) [inline knowledge-base/kb0059.md:29]: and here

```suggestion
  1. Right-click on the **Language Bar** and select **Settings**.
  2. Click Add, select **Icelandic**, expand the **Keyboards** node, and select **Show More**.
  3. Select **SIL IPA Unicode** from the expanded list of keyboards and click **OK**.
  4. Click **OK** again to save changes.  


- In Toolbox, configure your language settings:
   1. Click **Project**, **Language Encodings**
   2. Select the language you wish to link, and click **Modify**.
   3. Click **Options**, and in the **Keyboard** combo box, select **SIL IPA Unicode**
   4. Click **OK**, **Close** to save changes
```

File: knowledge-base/kb0060.md [hunk 13]
```diff
@@ -1,233 +1,90 @@
...
+| active page|REG_DWORD| 0 = On Screen Keyboard 1 = Character Map<br />2 = Entry Helper (not supported)3 = Keyboard Usage4 = Font Helper|
+| position|REG_SZ|4 comma separated numbers indicating the last pixel position of the On Screen Keyboard|
+| show toolbar|REG_DWORD|If 1, show the toolbar at the top of the On Screen Keyboard.&nbsp; Defaults to 1.&nbsp; This option can be changed by right-clicking on the On Screen Keyboard.|
+| HKCU\Software\Tavultesoft\Keyman Engine\7.0\On Screen Keyboard\Character Map |||
+| cell size|REG_DWORD| Height of each cell in the character map.&nbsp; This can be changed by dragging the slider at the bottom right of the character map.|
+| character|REG_DWORD| Last selected character in the character map.|
+| filter|REG_SZ| Filter string.&nbsp; This can be changed by clicking the Filter box in the Character Map.|
+| font|REG_SZ| Last selected display font for the Character Map.&nbsp; Can be changed by right-clicking the Character Map.|
+| insert mode|REG_DWORD| 0 = Code 1 = Character 2 = Name 3 = Text (default) 4 = Default 5 = Custom|
```
Comment (darcywong00) [inline knowledge-base/kb0060.md:38]: There's a few numbered lists in this table that might need newlines to show
```suggestion
| insert mode|REG_DWORD| 0 = Code<br/>1 = Character<br/>2 = Name<br/>3 = Text (default)<br/>4 = Default<br/>5 = Custom|
```

**Review by darcywong00 — APPROVED** [review]
lgtm

File: knowledge-base/kb0060.md [hunk 14]
```diff
@@ -1,233 +1,90 @@
...
+| quality|REG_DWORD|0 = Default Quality 1 = Draft Quality 2 = Proof Quality3 = Non Antialiased 4 = Antialiased 5 = Cleartype (default)|
```
Comment (Meng-Heng) [inline knowledge-base/kb0060.md:39]:
```suggestion
| quality|REG_DWORD|0 = Default Quality<br/>1 = Draft Quality<br/>2 = Proof Quality<br/>3 = Non Antialiased<br/>4 = Antialiased<br/>5 = Cleartype (default)|

```

File: knowledge-base/kb0060.md [hunk 15]
```diff
@@ -1,233 +1,91 @@
...
+| switch language with keyboard|REG_DWORD| If 1, switch the Windows language as appropriate when a Keyman Desktop keyboard is selected.&nbsp; This setting can be changed in Keyman Configuration.|
+| switch language for all applications|REG_DWORD| Not currently implemented|
+| enable hints|REG_DWORD| Enable hint messages throughout Keyman Desktop.&nbsp; This setting can be changed in Keyman Configuration.|
+| HKCU\Software\Tavultesoft\Keyman Engine\7.0\Diag |||
+| HKCU\Software\Tavultesoft\Keyman Engine\7.0\Hotkeys |||
+| HKCU\Software\Tavultesoft\Keyman Engine\7.0\On Screen Keyboard |||
+| show visual keyboard|REG_DWORD| If 1, displays the On Screen Keyboard when Keyman Desktop starts|
+| fade visual keyboard|REG_DWORD| If 1, makes the On Screen Keyboard partially transparent when the mouse cursor is not over it.&nbsp; This option can be changed by right-clicking on the On Screen Keyboard.|
+| active page|REG_DWORD| 0 = On Screen Keyboard 1 = Character Map<br />2 = Entry Helper (not supported)3 = Keyboard Usage4 = Font Helper|
```
Comment (Meng-Heng) [inline knowledge-base/kb0060.md:30]:
```suggestion
| active page|REG_DWORD| 0 = On Screen Keyboard 1 = Character Map<br />2 = Entry Helper (not supported)<br />3 = Keyboard Usage<br />4 = Font Helper|

```

File: knowledge-base/kb0060.md [hunk 16]
```diff
@@ -1,233 +1,91 @@
...
+| cell size|REG_DWORD| Height of each cell in the character map.&nbsp; This can be changed by dragging the slider at the bottom right of the character map.|
+| character|REG_DWORD| Last selected character in the character map.|
+| filter|REG_SZ| Filter string.&nbsp; This can be changed by clicking the Filter box in the Character Map.|
+| font|REG_SZ| Last selected display font for the Character Map.&nbsp; Can be changed by right-clicking the Character Map.|
+| insert mode|REG_DWORD| 0 = Code<br/>1 = Character<br/>2 = Name<br/>3 = Text (default)<br/>4 = Default<br/>5 = Custom|
+| quality|REG_DWORD|0 = Default Quality<br/>1 = Draft Quality<br/>2 = Proof Quality<br/>3 = Non Antialiased<br/>4 = Antialiased<br/>5 = Cleartype (default)|
+
+| HKLM\Software\Tavultesoft\Keyman Engine\7.0 |||
+| root path|REG_SZ| Installed path for Keyman Engine|
+| root keyboard admin path|REG_SZ| Path where Keyman Engine installs keyboard files for local machine.&nbsp; If left blank, Keyman Engine will install keyboard files in %CommonAppData%\Tavultesoft\KeymanEngine 7.0\Keyboard\|
```
Comment (Meng-Heng) [inline knowledge-base/kb0060.md:43]:
```suggestion
| root keyboard admin path|REG_SZ| Path where Keyman Engine installs keyboard files for local machine.&nbsp; If left blank, Keyman Engine will install keyboard files in %CommonAppData%\Tavultesoft\KeymanEngine 7.0\Keyboard\ |

```

File: knowledge-base/kb0060.md [hunk 17]
```diff
@@ -1,233 +1,91 @@
...
+
+# Tavultesoft Keyman keys
+
+The table below lists most of the common Keyman Desktop and Keyman Engine registry settings.  This is intended for use as reference only, and changes should not be made directly to these settings in most circumstances; instead use the Keyman Desktop Configuration tool to change settings.
+
+>
+||||
+|------|------|------|
+|HKCU\Software\Tavultesoft\Keyman Engine\7.0|
+| root keyboard user path|REG_SZ|Path where Keyman Engine installs keyboard files for current user.&nbsp; If left blank, Keyman Engine will install keyboard files in %AppData%\Tavultesoft\KeymanEngine 7.0\Keyboard\|
```
Comment (Meng-Heng) [inline knowledge-base/kb0060.md:14]:
```suggestion
| root keyboard user path|REG_SZ|Path where Keyman Engine installs keyboard files for current user.&nbsp; If left blank, Keyman Engine will install keyboard files in %AppData%\Tavultesoft\KeymanEngine 7.0\Keyboard\ |

```

**Pull request merged**